test/netgen.pkg.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
-
-# Copyright 2020-2021 Sai Charan Lanka <lankasaicharan123@gmail.com>
+# Authors:
+#   Unai Martinez-Corral
+#   Sai Charan Lanka
```

Comment:
You might keep your e-mail in the same line, if you want to.

Reply:
Nah! Its alright.